test/prototype_transforms_kernel_infos.py

```diff
@@ -278,26 +323,38 @@ def transform(bbox):
 
 
 def reference_inputs_affine_bounding_box():
-    for bounding_box_loader, angle, translate, scale, shear, center in itertools.product(
-        make_bounding_box_loaders(extra_dims=[(4,)], image_size=(32, 38), dtypes=[torch.float32]),
-        range(-90, 90, 56),
-        range(-10, 10, 8),
-        [0.77, 1.0, 1.27],
-        range(-15, 15, 8),
-        [None, (12, 14)],
+    for bounding_box_loader, affine_kwargs in itertools.product(
+        make_bounding_box_loaders(extra_dims=[()]),
+        _AFFINE_KWARGS,
     ):
         yield ArgsKwargs(
             bounding_box_loader,
             format=bounding_box_loader.format,
             image_size=bounding_box_loader.image_size,
-            angle=angle,
-            translate=(translate, translate),
-            scale=scale,
-            shear=(shear, shear),
-            center=center,
+            **affine_kwargs,
         )
 
 
+def sample_inputs_affine_image_mask():
+    for mask_loader, center in itertools.product(
+        make_mask_loaders(dtypes=[torch.uint8]),
+        [None, (0, 0)],
+    ):
+        yield ArgsKwargs(mask_loader, center=center, **_AFFINE_KWARGS[0])
+
+
+@pil_reference_wrapper
+def reference_affine_mask(*args, **kwargs):
+    return F.affine_image_pil(*args, interpolation=F.InterpolationMode.NEAREST, **kwargs)
```

Comment:
Previously we had a custom implementation for the reference (see below). @vfdev-5 was there a reason to not rely on PIL as reference here?

Reply:
The reason is that it was a correctness test and not consistency. We do not call our library to produce reference output. You can imagine a situation if both methods have changed but ci still passing.

Reply:
As discussed offline, Victor is technically right here. Still, since our PIL kernels, e.g. `affine_image_pil` mostly wrap the PIL functionaly and only juggle the parameters a bit, it is fine to use them directly as reference for now. We can always come back later and write proper reference tests.